WalletWasabi.Fluent/ViewModels/Wallets/CoinJoinStateViewModel.cs

```diff
@@ -89,7 +92,17 @@ public CoinJoinStateViewModel(WalletViewModel walletVm, IObservable<Unit> balanc
 
 		balanceChanged.Subscribe(_ => _stateMachine.Fire(Trigger.BalanceChanged));
 
-		PlayCommand = ReactiveCommand.Create(() => _stateMachine.Fire(Trigger.Play));
+		PlayCommand = ReactiveCommand.Create(async () =>
+		{
+			if (!_wallet.KeyManager.IsCoinjoinProfileSelected)
+			{
+				DialogResult<bool> isProfileSelected = await NavigateDialogAsync(new CoinJoinProfilesViewModel(_wallet.KeyManager, false), NavigationTarget.DialogScreen);
+			}
```

Comment:
I think this is not working. You press Play and Select the profile and nothing will happen. Did you test this?

Reply:
Tested, working. 
What can be improved, is to discard the result from the dialog, as it's not necessary to use it as `IsCoinjoinProfileSelected` will be set if the user chooses a profile this way.

Reply:
Changed it in latest commit